internal/resource/config/oauth/client/oauth_client_resource.go

```diff
@@ -1280,6 +1279,19 @@ func (r *oauthClientResource) ModifyPlan(ctx context.Context, req resource.Modif
 	}
 
 	planModified := false
+
+	if !pfVersionAtLeast120 &&
+		!pfVersionAtLeast121 &&
+		!pfVersionAtLeast122 {
+		if internaltypes.IsDefined(plan.PersistentGrantExpirationTime) {
+			plan.PersistentGrantExpirationTime = types.Int64PointerValue(plan.PersistentGrantExpirationTime.ValueInt64Pointer())
+		} else {
+			plan.PersistentGrantExpirationTime = types.Int64Null()
+		}
+	} else {
+		plan.PersistentGrantExpirationTime = types.Int64PointerValue(plan.PersistentGrantExpirationTime.ValueInt64Pointer())
```

Comment:
What is the purpose of this line? Is this trying to handle something related to unknowns?